### PR TITLE
Normalize nomenclature: marker row vs. delimiter row

### PR DESCRIPTION
**Pull request by waldyrious**
The code for the table extension used the term "marker row" for the line separating a table header from the body, but the spec calls it ["delimiter row"](https://github.github.com/gfm/#delimiter-row). This change normalizes the terminology in the code so that it's consistent with the spec.

This PR addresses issue #269.

### CODE REVIEW

**Comment by waldyrious** [comment]
As I've been seeing regular recent activity in the repo, and given this PR has been open for several months now, I thought I'd ping the people who have been pushing commits recently, for feedback on whether (and when, if so) this change will be considered: @phillmv, @anticomputer, @kevinbackhouse. I'd appreciate any input!

**Comment by kevinbackhouse** [comment]
I'm from the security team, and all of my activity has been bug-fixing. I have no opinion on this marker-vs-delimiter naming question. The code change looks harmless to me, so I'm fine with approving it if people think it's important.

**Review by kevinbackhouse — APPROVED** [review]

**Pull request merged**
